pom.xml

```diff
@@ -2,13 +2,19 @@
     <modelVersion>4.0.0</modelVersion>
     <groupId>org.neo4j</groupId>
     <artifactId>neo4j-jdbc</artifactId>
-    <version>2.3-SNAPSHOT</version>
+    <version>2.0.5-SNAPSHOT</version>
```

Comment:
Personally, I think I'd prefer simply increasing the version from the latest release and documenting it restored compatibility with earlier Neo4j versions. The version numbers follow the chronology at the moment.